temoa/data_processing/db_to_excel.py

```
@@ -0,0 +1,280 @@
+    query_all_techs = """
+        SELECT DISTINCT Efficiency.region, Efficiency.tech, Technology.sector
+        FROM Efficiency
+        INNER JOIN Technology ON Efficiency.tech = Technology.tech
+    """
+    all_techs = pd.read_sql_query(query_all_techs, con)
+
+    query_capacity = f"""
+        SELECT region, tech, sector, period, SUM(capacity) as capacity
+        FROM OutputNetCapacity WHERE scenario = '{scenario_name}'
+        GROUP BY region, tech, sector, period
+    """
```

Ruff (0.14.2)

Possible SQL injection vector through string-based query construction

(S608)